cdt_slugs.yaml

```diff
@@ -228,6 +228,8 @@ libnl-devel:
   # generates libnl too
   custom: false
   license_file: licenses/libnl-devel-license
+libnl3-cli:
+  custom: false
```

Comment:
@jakirkham I think in your PR you forgot to add the cdt to the slugs file. CDTs can be generated as deps of other cdts or explicitly requested. To explicitly request it, you have to add it to the slugs file.

Reply:
Yeah I tried this locally, but was still missing the CentOS 7 CDTs for thing like `libnl3`. Still not sure why that was happening.